Assets/UI/CQUICommon.lua

```diff
@@ -92,6 +94,34 @@ function CQUI_GetRealHousingFromImprovements(pCity:table)
     return pCity:GetGrowth():GetHousingFromImprovements() + Round(iNumHousing-math.floor(iNumHousing),1);
 end
 
+-- ===========================================================================
```

Comment:
this is the single function that was in supportfunctions.lua that CQUI referenced.
The one reference to this function was in a file that already includes CQUICommon.lua